client/src/components/QA/QACardAnswers.jsx

```diff
@@ -42,67 +42,25 @@ const QACardAnswers = ({ currentAnswers }) => {
-          </p>
-        </div>
-      )}
-      <button data-testid="qa-load-more" className="qa-load-more" onClick={() => {
-        setAnswersToShowLength(answersToShowLength + 2);
-        setAnswersToShow(answers.slice(0, answersToShowLength));
-      }}><b><span className="qa-load">[LOAD MORE ANSWERS]</span></b></button>
-    </div>
-  }
-
-  // if answers array is at max, show "collapse button"
-  if (answersLength < (answersToShowLength - 2)) {
-    return <div data-testid="qa-div-card-answers">
-      {answersToShow.map((answer, index) =>
-        <div className="qa-answers-external" key={index}>
-          <div className="qa-div-answers" ><b>A:</b> {answer.body}</div>
-          <p className="qa-footer"> by {answer.answerer_name}, {format((new Date(answer.date)), "MMMM dd, yyyy")}
-            <span className="qa-divider">|</span>
-            <span onClick={() => APICall(answer.id)}>Helpful? ({answer.helpfulness})</span>
-            <span className="qa-divider">|</span>
-            <span onClick={() => APICallReport(answer.id)}>Report</span>
-          </p>
-        </div>
-      )}
-      <button data-testid="qa-load-more" className="qa-load-more" onClick={() => {
-        setAnswersToShow(answers.slice(0, 2));
-        setAnswersToShowLength(2);
-      }}><b><span className="qa-load">[COLLAPSE ANSWERS]</span></b></button>
-    </div>
-  }
-
-  // if there are two or less answers, don't show any buttons
-  if (answersLength <= (answersToShowLength - 2)) {
-    return <div data-testid="qa-div-card-answers">
-      {answersToShow.map((answer, index) =>
-        <div className="qa-answers-external" key={index}>
-          <div className="qa-div-answers" ><b>A:</b> {answer.body}</div>
-          <p className="qa-footer"> by {answer.answerer_name}, {format((new Date(answer.date)), "MMMM dd, yyyy")}
-            <span className="qa-divider">|</span>
-            <span onClick={() => APICall(answer.id)}>Helpful? ({answer.helpfulness})</span>
-            <span className="qa-divider">|</span>
-            <span onClick={() => APICallReport(answer.id)}>Report</span>
-          </p>
-        </div>
-      )}
+  return (
+    <div data-testid="qa-answers-div">
+      <div data-testid="qa-div-card-answers">
+        {answersToShow.map((answer, index) =>
+          <div className="qa-answers-external" key={index}>
+            <div className="qa-div-answers" ><b>A:</b> {answer.body}</div>
+            <p className="qa-footer"> by {answer.answerer_name}, {format((new Date(answer.date)), "MMMM dd, yyyy")}
+              <span className="qa-divider">|</span>
+              <span onClick={() => APICallHelpful(answer.id)}>Helpful? ({answer.helpfulness})</span>
+              <span className="qa-divider">|</span>
+              <span onClick={() => APICallReport(answer.id)}>Report</span>
+            </p>
+          </div>
+        )}
+        <button data-testid="qa-load-more" className="qa-load-more" onClick={() => {
```

Comment:
For the conditional rendering, you could use an `&&` after your boolean condition:

```javascript
{answersLength <= (answersToShowLength - 2) && (
  <button ...
)}
```